oxygen-addon.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xt:extensions xmlns:xi="http://www.w3.org/2001/XInclude"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:xt="http://www.oxygenxml.com/ns/extension"
+	xsi:schemaLocation="http://www.oxygenxml.com/ns/extension http://www.oxygenxml.com/ns/extension/extensions.xsd">
+
+	<!-- This @id value assumes that https://github.com/xspec/ community owns "xspec.io". -->
+	<xt:extension id="io.xspec.xspec">
```

Comment:
It's intentional that the `id` value is duplicated on each `xt:extension` element, right?

Reply:
Yes, intentional, per this [explanation](https://www.oxygenxml.com/forum/topic14235.html#p41951):

>  the framework ID matters most if you want to install let's say a framework with an ID and Oxygen already finds a framework with the same ID installed (even if it was installed from another update site).

> You cannot have two add-ons with the same ID installed at the same time in an Oxygen version. It also does not make sense to allow for this.

Reply:
Great.